Formula/terragrunt.rb

```diff
@@ -0,0 +1,50 @@
+require "language/go"
+
+class Terragrunt < Formula
+  desc "Thin wrapper for Terraform e.g. for locking state."
+  homepage "https://github.com/gruntwork-io/terragrunt"
+  url "https://github.com/gruntwork-io/terragrunt.git",
```

Comment:
Does this use submodules? Otherwise, please use a tarball rather than a git checkout.